Projects-2017-2018/StringSortingLSD-MSDbyBuseDragomirAlexandru/LSD MSD string sorts/main.cpp

```diff
@@ -0,0 +1,59 @@
+#include <iostream>
+#include <vector>
+#include <algorithm>
+#include <string.h>
+#include "Element.h"
+#include "key_indexed_counting.h"
+
+using namespace std;
+
+int main()
+{
+    vector<Element> students;
+
+    Element *student1 = new Element("Andreson", 2);

```

Comment:
1. I find hardcoding the data pretty hard to maintain and prone to bugs. Let's say one wants to insert more data or different values. With the current approach, this can be done by modifying the source code and afterward rebuild the application. I recommend a more flexible approach: read the data from a file.

2. You are allocating memory with "new" but I see that you do not delete the allocated memory when you finish using it. This causes memory leaks. This can be resolved in three ways: 
- do not allocate memory with new (I see that you do not actually need to do it)
- use delete when you finish using the resource
- use unique_ptr to hold your element, more information about unique_ptr can be found here: http://en.cppreference.com/w/cpp/memory/unique_ptr